### PR TITLE
Updated Wadl2Checker's version string, some formatting, and several items identified by the static analysis tools.

### PR DESCRIPTION
**Pull request by wdschei**

### CODE REVIEW

File: cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala [hunk 1]
```diff
@@ -25,7 +25,14 @@ import org.clapper.argot.ArgotConverters._
...
 object Wadl2Checker {
-  val parser = new ArgotParser("wadl2checker", preUsage=Some("wadl2checker: Version 1.0.0-SNAPSHOT"))
+
+  val creatorString = {
+    val title = getClass.getPackage.getImplementationTitle
+    val version = getClass.getPackage.getImplementationVersion
+    s"$title ($version)"
+  }
+
+  val parser = new ArgotParser("wadl2checker", preUsage=Some(creatorString))
```
Comment (wdschei) [inline cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala]: Is this right or does this `creatorString` replace the `ArgotParser` completely?

**Comment by wdschei** [comment]
The new `--version` option and pre-usage will print something like:
```
   API Checker CLI WADL2Checker v1.1.3
```

**Comment by RackerWilliams** [comment]
@wdschei  This is awesome.  Can you do the same thing to wadl2dot? 😃

**Comment by wdschei** [comment]
@RackerWilliams Sure. This PR or another one?

File: cli/pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>com.rackspace.papi.components</groupId>
         <artifactId>api-checker</artifactId>
-        <version>1.1.2-SNAPSHOT</version>
+        <version>1.1.3-SNAPSHOT</version>
```
Comment (RackerWilliams) [inline cli/pom.xml]: What is  this about.  Weird.

Comment (wdschei) [inline cli/pom.xml]: I had a bad rebase/merge/squash.
It is fixed now.

**Pull request merged**